tools/azure-sdk-tools/ci_tools/environment_exclusions.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import logging
+
+PYLINT_ACCEPTABLE_FAILURES = [
```

Comment:
I'm wondering if we should change this name to PYLINT_OPT_OUT to match the other lists

Reply:
Absolutely! Done.